### ISSUE TITLE
Broken: gitkraken 2.4.0-1

### ISSUE THREAD

**Issue by EvanWashkow** [Antergos/antergos-packages#185]
Just ran an update today (April 24, 2017). Among other things, there was an upgrade for GitKraken, to v 2.4.0-1. However this version doesn't start at all. Attempting to open it in the terminal results in "Segmentation fault." Removing user configuration files doesn't fix.

System build:
Gnome 3.24.1 (just updated today along with GitKraken)
i7 x64, 16 GB RAM

**Comment by EvanWashkow** [comment]
@lots0logs It looks like you were the one who last updated this. GitKraken is important for my work.

**Comment by lots0logs** [comment]
Could you try launching it from terminal and post the output here?

**Comment by EvanWashkow** [comment]
^ I already did. 

`Attempting to open it in the terminal results in "Segmentation fault."`

**Comment by lots0logs** [comment]
There must have been some output other than that, no?

**Comment by EvanWashkow** [comment]
```
Node started time: 1493159480814
Segmentation fault (core dumped)
```

**Comment by EvanWashkow** [comment]
That's it. Is there a place I can look to find the log for the core dump?

**Comment by lots0logs** [comment]
It looks like gitkraken is not compatible with the version of curl that we now use in Arch. There is an AUR package you could install to work-around the issue: `libcurl-openssl-1.0 ` I recommend reporting this issue to gitkraken devs as its not a packaging problem.

**Comment by lots0logs** [comment]
@EvanWashkow I realize now that what I said earlier sounds ridiculous without some additional context :sweat_smile: The reason I said it wasn't a packaging issue is because since Gitkraken is an electron app, it should not be affected by the library versions on the local system. Its supposed to come bundled with everything it needs to run. So that makes the issue a bug, IMO. Anyway, I found an easy workaround that doesn't involve installing an AUR package so I patched the package in our repo :smiley:
